glide.yaml

```diff
@@ -43,3 +43,4 @@ import:
   - tools/clientcmd
 testImport:
 - package: github.com/onsi/gomega
+- package: github.com/onsi/ginkgo/ginkgo
```

Comment:
to run the tests

https://github.com/projectcalico/libcalico-go/blob/master/run-uts#L7